modules/slo/iam.tf

```diff
@@ -63,3 +71,10 @@ resource "google_storage_bucket_iam_member" "object-viewer" {
   role   = "roles/storage.objectViewer"
   member = "serviceAccount:${local.sa_email}"
 }
+
+resource "google_storage_bucket_iam_member" "object-viewer-legacy" {
```

Comment:
Why is the legacy role needed?

Reply:
I noticed errors in the logs that the service account didn't have `storage.bucket.get`:
```
google.api_core.exceptions.Forbidden: 403 GET https://storage.googleapis.com/storage/v1/b/rnm-slo-repo-dev?projection=noAcl&prettyPrint=false: slo-ops@rnm-shared-monitoring.iam.gserviceaccount.com does not have storage.buckets.get access to the Google Cloud Storage bucket.
```

Reply:
@morgante should we keep as is or create a custom role with just 1 permission ?

Reply:
That is odd. I think this is fine as-is then.